Electron/app/main.ts

```diff
@@ -260,7 +260,24 @@ const installTrans = () => {
     }
     translations = data as { [key: string]: string };
     if (win != null) {
-      win.setTitle(translations['PAGES.HOME.TITLE'] || '')
+      interface PackageJson {
+        version: string;
+      }
+      let packageJson: PackageJson = {} as PackageJson;
+      let version = '';
+      let WinTitle = translations['PAGES.HOME.TITLE'];
+      let WinTitle_err = '';
+      fs.readFile(path.join(__dirname, 'package.json'), 'utf8', (err, data) => {
```

Comment:
package.json is an internal src file, it won't be available in the release artifact.

Reply:
I asked AI about this and they said it's feasible. I suggested trying compiling it, but I didn't compile it locally...........